odk1-src/central-projects.rst

```diff
@@ -125,6 +125,33 @@ Managing Project App Users
 
 To manage App Users for a Project, you can navigate to the Project overview page, then click on the :guilabel:`App Users` tab under the Project name. For more information about creating, managing, and retiring Project App Users, please see :ref:`this section <central-users-app-overview>`.
 
+.. _central-projects-form-workflow:
+
+Managing Project Form Workflow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Right now, Central offers two features around Form Workflow:
+
+ - Each Form's :ref:`Lifecycle State <central-forms-lifecycle>` controls whether App Users can download and/or submit to that Form. Near the end of a Form's life, for example, it makes sense to disallow downloading the Form as a blank, but still receive any submissions that have already been created.
+ - Access to download and submit each Form can be customized per App User associated with the Project. When first creating a Form, for example, it makes sense to only allow a testing user access to the Form so that one can be sure that it works before rolling it out to all users.
+
+In the future, additional workflow features are planned to make processes like training and drafting Forms easier.
```

Comment:
s/drafting/testing

Reply:
what.. how did you even make this comment? this line of text hasn't existed in a long time. :)